Src/Routes/UserRoute.ts

```diff
@@ -0,0 +1,16 @@
+import express, { Router } from "express";
+import { createUser } from "../Controllers/UserController";
+import { validateUser } from "../Middlewares/UserMiddleware";
+import { validateRequiredParams } from "../Middlewares/ValidatorMiddleware";
+
+const router = express.Router();
+
+router
+	.route("/")
+	.post(
+		validateRequiredParams(["firstName", "lastName", "email", "password"]),
```

Comment:
I was able to bypass this validator by using empty strings.

<img width="1321" alt="Screenshot 2023-04-08 at 9 37 48 AM" src="https://user-images.githubusercontent.com/30635021/230724526-8e860f5c-e09c-43bf-b2ca-3d78cd94e41d.png">